tests/nvmeof/test_ceph_nvmeof_alerts_events.py

```diff
@@ -427,10 +428,164 @@ def check_warning_status(gw, alert=True):
     )
 
 
+def test_ceph_83611098(ceph_cluster, config):
+    """[CEPH-83611098] Warning at Ceph cluster on zero listener in a subsystem.
+
+    NVMeoFZeroListenerSubsystem alert helps user to add listeners to the subsystem,
+    otherwise it would be not visible and able to connect from initators.
+
+    Args:
+        ceph_cluster: Ceph cluster object
+        config: test case config
+    """
+
+    time_to_fire = 600
```

Comment:
can we consider passing these via config for flexibility across environments. WDYT?